base/sys_info_starboard.cc

```diff
@@ -56,4 +58,17 @@ int64_t SysInfo::AmountOfVirtualMemory() {
   return AmountOfPhysicalMemoryImpl();
 }
 
+// static
+std::string SysInfo::OperatingSystemName() {
```

Comment:
This and below previously just weren't defined/ever used, right?

Reply:
Ya, I got the original code working by removing the metrics code that calls these methods in the original PR. It turns out the unit tests really want these, so instead of commenting out a bunch of unit tests, I just added a quick implementation here. It's worth mentioning, we don't use these values in our telemetry payloads atm, so it's a no-op from that pov.